app/models/event.rb

```diff
@@ -12,6 +12,16 @@ def self.active(early_access: false)
     end
   end
 
+  def low_income_open
```

Comment:
tiny thing, but ruby convention would be to have a method that returns true or false to be `low_income_open?`

Reply:
Happy to change it